styles/globalStyle.js

```diff
@@ -119,27 +119,44 @@ const GlobalStyle = createGlobalStyle`
       padding: 1rem;
     }
 
-    &.input-form
+    &.input-form {
+
       input, textarea {
-      margin-top: 1rem;
-      border: 1px solid ${({ theme }) => theme.colors.mediumGray};
-      background-color: ${({ theme }) => theme.colors.mediumLightGray};
+        margin-top: 0.75rem;
+        border: 1px solid ${({ theme }) => theme.colors.mediumGray};
+        background-color: ${({ theme }) => theme.colors.mediumLightGray};
 
-      &:focus {
-        outline: ${({ theme }) => theme.colors.thatBlue} auto 5px;
-      }
+        &:focus {
+          outline: ${({ theme }) => theme.colors.thatBlue} auto 1px;
+        }
+
+        &.invalid {
+          border-color: ${({ theme }) => theme.colors.danger};
+
+          &:focus {
+            outline-offset: 0;
+            outline: unset;
+            border-color: ${({ theme }) => theme.colors.danger};
+            box-shadow: 0 0 0 0.2rem rgba(220, 53, 69, 0.55);
+          }
+        } //invalid
 
-      &.invalid {
-        border-color: ${({ theme }) => theme.colors.danger};
+      } //input, textarea
+
+      .react-select-container.invalid {
+        border: 1px solid red;
 
         &:focus {
-          outline: unset;
-          border-color: ${({ theme }) => theme.colors.danger};
-          box-shadow: 0 0 0 0.2rem rgba(220, 53, 69, 0.55);
-        }
+            outline-offset: 0;
+            outline: unset;
+            border-color: ${({ theme }) => theme.colors.danger};
+            box-shadow: 0 0 0 0.2rem rgba(220, 53, 69, 0.55);
```

Comment:
is this a color we need to put in the theme as well?

Reply:
it is in there in hex, it is warning color, but I need to set the semi-transparency